nextcloud_install_production.sh

```diff
@@ -389,6 +389,8 @@ crontab -u www-data -l | { cat; echo "*/5  *  *  *  * php -f $NCPATH/cron.php >
 
 # Run the updatenotification on a schelude
 occ_command config:system:set upgrade.disable-web --value="true"
+occ_command app:disable updatenotification
```

Comment:
@enoch85 this is unneccessary, because it gets restored at least after every major update.
we could also remove it after every update, but maybe it is enough to just get disabled. What do you think?